mobile_verifier/src/reward_shares.rs

```diff
@@ -475,11 +482,14 @@ impl CoverageShares {
                 })
                 .collect();
 
-            let oracle_boosting_status = if banned_radios.contains(&pubkey, cbsd_id.as_deref()) {
-                OracleBoostingStatus::Banned
-            } else {
-                OracleBoostingStatus::Eligible
-            };
+            let oracle_boosting_status =
+                if unique_connections::is_qualified(unique_connections, &pubkey, &radio_type) {
+                    OracleBoostingStatus::Qualified
```

Comment:
a qualified hotspot cannot be banned ?

Reply:
That is correct.

https://github.com/helium/HIP/blob/main/0134-reward-mobile-carrier-offload-hotspots.md#detailed-explanation

> Qualified Hotspots earn full PoC rewards regardless of whether they have a call detail record (CDR) or verification mapping event from a discovery mapper. For example, a qualified Hotspot would not have to meet the requirements of HIP 131, HIP 133, or similiar location based HIPs to earn full PoC. Applies to outdoor and indoor wifi hotspots.